### DISCUSSION TITLE
Integrazione alexa

### DISCUSSION THREAD

**Discussion by Sterobello** [indomus/forum#3333] [HUB: Home Assistant]
Ciao sto seguendo la guida "integrare alexa in HA" sono fermo all'inizio quando digito https://il_tuo_dominio:porta/api corrispondente al mio indirizzo (https://******.duckdns.org:8123/api mi viene restituito il seguente messaggio 404: Not Found. cosa posso fare? ovviamente ho seguito più volte tutti i passaggi senza successo.

**Comment by snakuzzo** [comment]
Però mamma mia...non cominciamo a creare topic a ca**o per ogni cavolata. Devi leggere bene la guida e quello che c'è scritto.

La URL che ti interessa e che deve restituire 405 è `/api/alexa/smart_home`.
Invece, la URL `/api/alexa/smart_home/api` è normale che ti restituisca 404. Nella guida c'è scritto chiaramente che servirà per successive configurazione (lato AWS)...e tieni presente che restituirà SEMPRE 404.

Fine
